packages/http/src/utils.ts

```diff
@@ -50,6 +50,6 @@ export function parseGraphQLBody(
       }
     default:
       // If no Content-Type header matches, parse nothing.
-      return null;
```

Comment:
I think it is  meaning-wise better for `graphqlHTTP` to also accept `body` as `null`  since it would explicitly be saying the body is empty. Wdyt?

Reply:
I recently read this: https://basarat.gitbook.io/typescript/recap/null-undefined, a few excerpts:

> JavaScript (and by extension TypeScript) has two bottom types : `null` and `undefined`. They are intended to mean different things:
> * Something hasn't been initialized : `undefined`.
> * Something is currently unavailable: `null`.

> TypeScript team doesn't use `null` : [TypeScript coding guidelines](https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#null-and-undefined) and it hasn't caused any problems. Douglas Crockford thinks [`null` is a bad idea](https://www.youtube.com/watch?v=PSGEjv3Tqo0&feature=youtu.be&t=9m21s) and we should all just use `undefined`.

> However, NodeJS style code bases uses `null` for Error arguments as standard as it denotes `Something is currently unavailable`. I personally don't care to distinguish between the two as most projects use libraries with differing opinions and just rule out both with `== null`.

On one hand, `null` would better convey the meaning of "body is not available", on the other hand the TypeScript team doesn't use it... 😅 I don't have a strong opinion about this, but I think I would keep it as I submitted. But I don't mind switching things around and add `null` as `body` type on `HTTPRequest`, either option is fine with me.

You pick, and I'll make the change if necessary 🙂

Reply:
In my opinion I would probably go with `null`, since `undefined` to me is all about a value that has not been assigned a value, whereas `null` explicitly calls out the absence of the object (in this case the body is absent).

http://www.ecma-international.org/ecma-262/6.0/#sec-terms-and-definitions-undefined-type

> 4.3.10 undefined value
> primitive value used when a variable has not been assigned a value
> 4.3.12 null value
> primitive value that represents the intentional absence of any object value

On a side note though, I do wish JS did not invent both `null` and `undefined` :p

Reply:
I'll make the change soon :)

Reply:
@hoangvvo Can you please review again?